tests/e2e/csi_snapshot_basic.go

```diff
@@ -105,6 +114,13 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			c = remoteC
 		}
+
```

Comment:
Can we try to collect events from SV namespace, so that if any provisioning of pvc fails or snapshot creation, we will have events from sv namespace as well